spec/features/consumer/shopping/unit_price_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+feature "As a consumer, I want to check unit price information for a product ;", js: true do
+  include AuthenticationHelper
+  include WebHelper
+  include ShopWorkflow
+  include UIComponentHelper
+
+  let(:distributor) { create(:distributor_enterprise, with_payment_and_shipping: true) }
+  let(:supplier) { create(:supplier_enterprise) }
+  let(:oc1) { create(:simple_order_cycle, distributors: [distributor], coordinator: create(:distributor_enterprise), orders_close_at: 2.days.from_now) }
+  let(:product) { create(:simple_product, supplier: supplier) }
+  let(:variant) { product.variants.first }
+  let(:order) { create(:order, distributor: distributor) }
+  let(:exchange1) { oc1.exchanges.to_enterprises(distributor).outgoing.first }
+  let(:user) { create(:user, password: "password", password_confirmation: "password") }
+
+  before do
+    set_order order
+    exchange1.update_attribute :pickup_time, "monday"
+    add_variant_to_order_cycle(exchange1, variant)
+    allow(OpenFoodNetwork::FeatureToggle)
+      .to receive(:enabled?).with(:unit_price, anything) { true }
+  end
+
+  describe "for the shopfront" do
+    before do
+      visit shop_path
+    end
+    
+    it "shows the unit price information with the question mark icon." do
```

Comment:
The dot at the end is uncommon in our codebase but I wonder if that makes for better reading of the test output. :thinking:

Reply:
I just removed it.